app/Services/OfferService.php

```
@@ -0,0 +1,118 @@
+                    'thumbnail_url' => $offer->getFirstMediaUrl('images', 'thumb'),
+                    'favorites_count' => $offer->favorites()->count(),
+                    'favorited_by_user' => $user ? $offer->favorites()->where('user_id', $user->id)->exists() : false,
+                    'condition' => $offer->getConditionEnum()?->label(),
+                    'conditionNumber' => $offer->condition,
+                    'status' => $offer->getStatusEnum()?->label(),
+                    'statusNumber' => $offer->status,
+                ];
+            });
+    }
+
+    public function getOfferDetail(Offer $offer, User|null $user = null, string $langColumn): Offer
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (Unused Code Rules)

(UnusedFormalParameter)